include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -350,11 +352,13 @@ using AllocatedStringPtr = std::unique_ptr<char, detail::AllocatedFree>;
  */
 struct Status : detail::Base<OrtStatus> {
```

Comment:
>struct Statu [](http://example.com/codeflow?start=0&length=12)

All constructors seem to be noexcept

Reply:
I updated all `Ort::Status` constructors to noexcept.

Reply:
I am not sure they all can be noexcept. If so, we want to make sure no exception can escape from there.